pkg/extensions/search/common/common_test.go

```diff
@@ -2020,7 +2020,11 @@ func TestBuildImageInfo(t *testing.T) {
 		imageConfig, err := olu.GetImageConfigInfo(invalid, manifestDigest)
 		So(err, ShouldBeNil)
 
-		imageSummary := search.BuildImageInfo(invalid, invalid, manifestDigest, manifest, imageConfig)
+		// olu.CheckManifestSignature(invalid, manifestDigest).
```

Comment:
Decide on one way to get signature or assume it is unsigned, and clean this code.